src/frontends/tensorflow_common/src/op/sparse_tensor_dense_add.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/scatter_nd_update.hpp"
+#include "utils.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_sparse_tensor_dense_add_op(const NodeContext& node) {
+    default_op_checks(node, 4, {"SparseTensorDenseAdd"});
+    auto a_indices = node.get_input(0);
+    auto a_values = node.get_input(1);
+    auto a_shape = node.get_input(2);
+    auto b = node.get_input(3);
+
+    // create dense tensor
+    auto zero_const = create_same_type_const_scalar<int32_t>(a_values, 0);
+    ov::Output<ov::Node> a = make_shared<v3::Broadcast>(zero_const, a_shape);
+    a = make_shared<v15::ScatterNDUpdate>(a, a_indices, a_values);
```

Comment:
great job. I propose you to move this code into `utils` (https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tensorflow_common/include/utils.hpp and cpp) and re-use it by your translator and `SparseTensorDenseMatMul`. How does it sound for you?

Reply:
I thought about it. Thank you for your quick reply. I'll update it right away.